requirements/dev.txt

```diff
@@ -93,6 +94,7 @@ slumber==0.7.1            # via edx-rest-api-client
 snowballstemmer==1.2.1    # via pydocstyle
 stevedore==1.28.0         # via edx-opaque-keys
 testfixtures==6.2.0
+tincan==0.0.5
```

Comment:
@georgebabey I discussed this with the team here and we think the that implementing a client to talk to LRS is not very difficult but the main problem is serializing data into xAPI format. It would require a lot of work and testing to get everything light and would be at least a 13 point story. 
Our suggestion is that we adopt tincan API, fork the repo and make our updates there, there will be a learning curve for the existing code base but it will be relative simpler as compared to creating xAPI serializer from scratch. Thoughts?